src/components/InLineNotification.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react'
+import { GestureResponderEvent, StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import AttentionIcon from 'src/icons/Attention'
+import Colors from 'src/styles/colors'
+import fontStyles, { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+export enum Severity {
+  Informational,
+  Warning,
+  Error,
+}
+
+interface Props {
+  severity: Severity
+  title?: string | null
+  description: string
+  style?: StyleProp<ViewStyle>
+  ctaLabel?: string | null
+  onPressCta?: () => void
+  ctaLabel2?: string | null
+  onPressCta2?: () => void
+  testID?: string
+}
+
+interface CustomStyles {
+  primary: Colors
+  secondary: Colors
+}
+
+export function InLineNotification({
+  severity,
+  title,
+  description,
+  style,
+  ctaLabel,
+  onPressCta,
+  ctaLabel2,
+  onPressCta2,
+  testID,
+}: Props) {
+  const severityStyle = severityStyles[severity]
+  const renderCtaLabel = (
+    label?: string | null,
+    onPress?: (event: GestureResponderEvent) => void,
```

Comment:
Is this the same `onPress` we get from top-level component props? I'm confused why the typings are different.

Reply:
It comes from `node_modules/react-native/Libraries/Text/Text.d.ts`
```ts
onPress?: ((event: GestureResponderEvent) => void) | undefined;
```

Reply:
I mean, why the typings for `onPressCta`/`onPressCta2` are different from this one?

Reply:
I've edited my answer above to clarify, basically that's how `Warning` component was. While `Text` component provides `GestureResponderEvent` as argument, we don't really need it, nor use it. Previously `onPressCta` would ignore `event`.

I've updated the code now to match the typing from `Text`, so we can process `GestureResponderEvent` parameter in a function passed to `onPressCta` should we need to in the future.

Reply:
we could also have `onPress?: () => void`, right?